web/src/components/molecules/Common/Form/AssetItem/index.tsx

```diff
@@ -79,7 +83,7 @@ const AssetItem: React.FC<Props> = ({
     maxCount: 1,
     directory: false,
     showUploadList: true,
-    accept: imageFormats + "," + fileFormats,
+    accept: imageFormats + "," + fileFormats + "," + compressedFileFormats,
```

Comment:
Can you move the accepted formats in to a variable and put it outside of the functional component since it won't ever change. Something like:
```
const acceptedFormates = `${imageFormats}, ${fileFormats}, ${compressedFileFormats}`

Reply:
Done!